### DISCUSSION TITLE
Praposal to updating Hindi Translation Language of OJS

### DISCUSSION THREAD

**Discussion by shantanu198713** [pkp/pkp-lib#8988] [Proposals]
Hi @asmecher @jonasraoni 

We are using OJS [https://epatrika.icar.org.in](https://epatrika.icar.org.in)  in full translation of `Hindi Language` and I would like to prapose to upgrade Hindi translation of the OJS.

Here I am mentioning [URL](https://github.com/shantanu198713/OJS-hi_IN)  for translation file code.

**Comment by asmecher** [comment]
Hi @shantanu198713! Was this translation created against the 3.3.0-x .po files, or the `main` branch currently in Github?

**Reply by shantanu198713** [reply]
Hi @asmecher 
Yes, This .po file use with OJS-3.3.0-14 and if required I will update with OJS-3.4. The file I have mentioned in the link above is the same that I am using in the OJS-3.3.0-14.

**Reply by asmecher** [reply]
Thanks, @shantanu198713! I think the best approach would be to add the translation to the `stable-3_3_0` branch (for the next 3.3.0-x release), and also to the `main` branch (for release in OJS 3.4.0). Once it's added, I'd recommend maintaining it by using the Weblate tool (https://translate.pkp.sfu.ca/); that way it's possible for multiple translators to collaborate on the work.

The files you've added appear to be from `locale/hi_ID`; there are numerous other translation files for a complete translation, and I'm pretty sure you'll have some of the other ones translated too. A complete translation should include:
- `lib/pkp/locale/hi_ID/*.po`
- `locale/hi_ID/*.po`
- `lib/pkp/plugins/*/*/locale/hi_ID/*.po`
- `plugins/*/*/locale/hi_ID/*.po`

Could you make sure all completed translations are available for addition?

**Reply by shantanu198713** [reply]
Thanks for reply @asmecher 
I am not friendly with Weblet but I will try to work with it. Some `Plugins` are translated as per our requirements like `Quick Submit Plugin`, and `Crossref Plugin`.

**Reply by asmecher** [reply]
Thanks, @shantanu198713! The most important thing for moving forward is making sure that I've got a copy of *all* the completed translation files; I'm pretty sure you'll have some additional ones at least in `lib/pkp/locale/hi_ID/`.
